src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -314,6 +321,225 @@ export default class GroupComparisonStore extends ComparisonStore {
         },
     });
 
+    readonly mutations = remoteData({
+        await: () => [this.samples, this.mutationEnrichmentProfiles],
+        invoke: async () => {
+            if (
+                this.samples.result!.length === 0 ||
+                this.mutationEnrichmentProfiles.result!.length === 0 ||
+                !this.selectedMutationMapperGene
```

Comment:
@inodb what should the default gene by?  should we show something by default?  this is tied up with what genes we want to offer the user by default.